blocks/activity-list/index.js

```diff
@@ -1,83 +1,55 @@
+const { Fragment } = wp.element;
 const { __ } = wp.i18n;
-const { registerBlockType, InspectorControls } = wp.blocks;
-const { Placeholder, withAPIData, TextControl, PanelBody } = wp.components;
-const { RawHTML } = wp.element;
-const { decodeEntities } = wp.utils;
+
+const blockName = 'amp-travel/activity-list';
 
 /**
  * Register block.
  */
 export default registerBlockType(
-	'amp-travel/activity-list',
+	blockName,
 	{
-		title: __( 'Activity list' ),
+		title: __( 'Activity list', 'travel' ),
 		category: 'common',
 		icon: 'tickets',
 		keywords: [
-			__( 'Browse activities' ),
-			__( 'Travel' )
+			__( 'Browse activities', 'travel' ),
+			__( 'Travel', 'travel' )
 		],
 
-		edit: withAPIData( () => {
-			return {
-				activityResults: '/wp/v2/activities'
-			};
-		} )( ( { activityResults, isSelected, setAttributes, attributes: { heading } } ) => {
-			const hasActivities = Array.isArray( activityResults.data ) && activityResults.data.length;
-			if ( ! hasActivities ) {
-				return (
-					<Placeholder key="placeholder" icon="admin-post" label={ __( 'Activities' ) }
-					>
-						{ __( 'No activities found, add some to use the block.' ) }
-					</Placeholder>
-				);
-			}
-
-			const activities = activityResults.data;
+		edit( { attributes, isSelected, setAttributes } ) {
+			const { heading } = attributes;
 
 			return [
 				isSelected && (
-					<InspectorControls key='inspector'>
-						<PanelBody title={ __( 'Activity List settings' ) }>
+					<InspectorControls key="activity-inspector">
+						<PanelBody title={ __( 'Activity List settings', 'travel' ) }>
 							<TextControl
-								label={ __( 'Activity List Header' ) }
+								label={ __( 'Activity List Header', 'travel' ) }
 								value={ heading }
 								onChange={ ( value ) => setAttributes( { heading: value } ) }
 							/>
 						</PanelBody>
 					</InspectorControls>
 				),
-				<section key='activities' className='travel-activities pb4 pt3 relative'>
-					<div className='max-width-3 mx-auto px1 md-px2'>
-						<h3 className='center bold h1 line-height-2'>{ heading }</h3>
-					</div>
-					<div className='travel-overflow-container'>
-						<div className='justify-center flex flex-wrap flex-auto p1 md-px1 mxn1'>
-							{ activities.map( ( activity, i ) =>
-								<a key='activity' href={ activity.link } className={ 'travel-activities-activity mx1 travel-type-' + activity.slug } target="_blank">
-									<div className='travel-shadow circle inline-block'>
-										<div className='travel-activities-activity-icon'>
-											<RawHTML key='html'>{ decodeEntities( activity.svg ) }</RawHTML>
-											<RawHTML key='html'>{ decodeEntities( activity.meta.amp_travel_activity_svg ) }</RawHTML>
-										</div>
-									</div>
-									<p className='bold center line-height-4'>
-										{ activity.name }
-									</p>
-								</a>
-							) }
-						</div>
-					</div>
-				</section>
+
+				<Fragment key="activity-ssr">
```

Comment:
We could wrap this into `<Disabled>` instead to prevent interacting with the view and being directed away when clicking on links.

Reply:
Good idea, https://github.com/xwp/travel/pull/86/commits/4ce2b7fe1257973b51832b298045176606075d77 adds `<Disabled>`